src/containers/pages/FocusInvestigation/map/active/tests/index.test.tsx

```diff
@@ -348,4 +348,43 @@ describe('containers/pages/FocusInvestigation/activeMap', () => {
     expect(currentGoalMock).not.toBeCalled();
     expect(FCMock).not.toBeCalled();
   });
+
+  it('renders the GisidaWrapper with structures', () => {
+    const mock: any = jest.fn();
+    const supersetServiceMock: any = jest.fn(async () => []);
+    store.dispatch(fetchGoals([fixtures.goal3 as goalDucks.Goal]));
+    store.dispatch(fetchJurisdictions([fixtures.jurisdictions[0]]));
+    store.dispatch(fetchPlans([fixtures.plan1 as Plan]));
+    store.dispatch(fetchTasks(fixtures.tasks));
+    store.dispatch(structureDucks.setStructures(structures as structureDucks.Structure[]));
+
+    const props = {
+      currentGoal: fixtures.goal3,
+      history,
+      location: mock,
+      match: {
+        isExact: true,
+        params: { id: fixtures.plan1.id },
+        path: `${FI_SINGLE_URL}/:id`,
+        url: `${FI_SINGLE_URL}/13`,
+      },
+      pointFeatureCollection: wrapFeatureCollection([fixtures.coloredTasks.task3.geojson]),
+      polygonFeatureCollection: wrapFeatureCollection([fixtures.coloredTasks.task2.geojson]),
+      supersetService: supersetServiceMock,
+    };
+    const wrapper = mount(
+      <Provider store={store}>
+        <Router history={history}>
+          <ConnectedMapSingleFI {...props} />
+        </Router>
+      </Provider>
+    );
+
+    // structures prop
+    const singleActiveWrapperProps = wrapper.find('SingleActiveFIMap').props();
+    expect((singleActiveWrapperProps as MapSingleFIProps).plan).toEqual(fixtures.plan1);
+    expect((singleActiveWrapperProps as MapSingleFIProps).structures).not.toBeNull();
```

Comment:
@moshthepitt @p-netm @FrankApiyo - Please assist me here, I cannot figure out why these are failing...

Structures is null because the plan is null because for some reason the component isn't getting a Plan from the store when it maps state to props. I've [copied this test implementation from directly above]((singleActiveWrapperProps as MapSingleFIProps).plan) so I don't understand why `(singleActiveWrapperProps as MapSingleFIProps).plan` is `null`.